app/controllers/account/transactions_controller.rb

```diff
@@ -15,7 +15,7 @@ def update
     @transaction.update! transaction_params
     @transaction.sync_account_later
 
-    redirect_back_or_to account_transaction_url(@transaction.account, @transaction), notice: t(".success")
+    render turbo_stream: turbo_stream.replace(@transaction, partial: "transaction", locals: { transaction: @transaction })
```

Comment:
Could be wrong here, but I think we'd need to provide an HTML only response for this?

```rb
respond_to do |format|
  format.html { redirect_back_or_to account_transaction_path(@account, @transaction) }
  format.turbo_stream { render turbo_stream: turbo_stream.replace(@transaction) }
end
```

Reply:
Done!